packages/mdl-snackbar/_variables.scss

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Hard coded since the color is not present in any palette.
+$mdl-snackbar-background-color: #323232 !default;
```

Comment:
nit: you can remove default here. If we find that the colors need to be customized via sass in the future, we can always add it in.